homeassistant/components/tradfri/__init__.py

```diff
@@ -59,8 +52,8 @@
     {
         DOMAIN: vol.Schema(
```

Comment:
We normally just mark the domain with `cv.removed` and remove the rest of the schema.